File: Leetcode/2873.go
package Leetcode

func maximumTripletValue(nums []int) int64 {
	n := len(nums)
	var ans int64 = 0
	for i := 0; i < n; i++ {
		for j := i + 1; j < n; j++ {
			for k := j + 1; k < n; k++ {
				ans = max(ans, int64((nums[i]-nums[j])*nums[k]))
			}
		}
	}
	return ans
}

File: Leetcode/2873.py
from typing import List


class Solution:
    def maximumTripletValue(self, nums: List[int]) -> int:
        arr = nums
        n = len(arr)
        ans = 0
        for i in range(n):
            for j in range(i + 1, n):
                for k in range(j + 1, n):
                    ans = max(ans, (arr[i] - arr[j]) * arr[k])
        return ans
